<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindFriendWindow</class>
 <widget class="QDialog" name="FindFriendWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找好友</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>351</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="accountInfoGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>好友信息</string>
    </property>
    <widget class="QLabel" name="signatureLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="numLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nicknameLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="truenameLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sexLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ageLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="schoolLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加为好友</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>好友号码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="findNumLineEdit">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="findButton">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
